src/Service.Host/client/src/components/importBooks/ImportBook.js

```diff
@@ -354,11 +348,24 @@ function ImportBook({
             !state.impbook.pva ||
             !state.impbook.foreignCurrency ||
             `${calcRemainingTotal()}` !== '0' ||
-            `${calcRemainingDuty()}` !== '0' ||
-            `${calcRemainingWeight()}` !== '0'
+            `${calcRemainingDuty()}` !== '0'
         );
     };
 
+    const [dialogOpen, setDialogOpen] = useState(false);
+
+    const handleSaveClick = () => {
+        if (`${calcRemainingWeight()}` !== '0' && !dialogOpen) {
```

Comment:
why not calcRemainingWeight() !== 0
i.e. why toString both sides of the expression?

Reply:
Good question, I'll try it and see. I think I did it for safety cause of js comparisons